Source/Scene/Scene.js

```diff
@@ -539,6 +539,15 @@ define([
          */
         this.fog = new Fog();
 
+        /**
+         * Determines the uniform depth size in meters of each frustum of the multifrustum in 2D. If a primitive or model close
+         * to the surface shows z-fighting, decreasing this will eliminate the artifact, but decrease performance. On the
+         * other hand, increasing this will increase performance but may cause z-fighting among primitives close to thesurface.
+         * @type {Number}
+         * @default 1.75e6
+         */
+        this.frustumSize2D = 1.75e6;
```

Comment:
Put this under `farToNearRatio` and perhaps name it `nearToFarDistance2D` for better symmetry.

Update CHANGES.md.